docs/modules/string.md

```diff
@@ -124,6 +164,25 @@ _Example_
 
 ***
 
+## String.fromCharCode
```

Comment:
Not related to this PR, but shouldn't we use [fromCodePoint](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/fromCodePoint) and `codePointAt` instead, to handle higher code points as well? Both are available since at least node 4.2.6.